### ISSUE TITLE
Unable to remove "John Lennon" from "The Beatles"

### ISSUE THREAD

**Issue by rogerpueyo** [joaomlneto/antidote-rest-server#9]
I'm following the quick start guide, adding some people to "The Beatles":
```
$ curl localhost:3000/set/add/bands/beatles/john
ok
$ curl localhost:3000/set/add/bands/beatles/paul
ok
$ curl localhost:3000/set/add/bands/beatles/george
ok
$ curl localhost:3000/set/add/bands/beatles/ringo
ok
```

Everything seems to be fine:
```
$ curl localhost:3000/set/read/bands/beatles
["john","paul","ringo","george"]
```

However, when I try to remove John from the band, I get:
```
$ curl localhost:3000/set/remove/bands/beatles/john
curl: (52) Empty reply from server
```

And _antidote-rest-server_ says:
```
$ antidote-rest-server
Antidote REST Server v0.9.2
Using Antidote at localhost:8087
REST Server listening on port 3000
(node:4227) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): ReferenceError: defer is not defined
(node:4227) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

```

**Comment by rogerpueyo** [comment]
Fixed as of antidote-rest-server v0.9.3.

**Comment by joaomlneto** [comment]
fixed in 41f16b3819fb9402688bc7415f59ba2eb14952a6
